src/main/java/seedu/address/model/transaction/Description.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.model.transaction;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Transaction's Description in the Wallet.
+ * Guarantees: immutable; is valid as declared in {@link #isValidDescription(String)}
+ */
+public class Description {
```

Comment:
Is Description too generic of a class name? Because it can be a description for a person etc.

Reply:
but it's in the transaction package tho. 🤔

Reply:
Ya I think it’s fine ba cus inside package